spec/models/user_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user) { FactoryBot.create(:user) }
+  let(:invalid_user) { FactoryBot.create(:user) }
+  describe 'Model : ' do
+    describe 'check to be a valid ' do
+      it 'factory ' do
```

Comment:
any reason to leave extra space in the end of string?

Reply:
Bhaiya, I didn't notice when I wrote this. Thank you, Bhaiya, for suggesting me. I will fix this.